apps/scan/backend/package.json

```diff
@@ -99,7 +100,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-vx": "workspace:*",
     "is-ci-cli": "^2.1.2",
-    "jest": "^26.6.3",
+    "jest": "^27.0.0",
```

Comment:
This was because I ran into this issue that caused Jest worker processes not to exit when the tests were done: https://stackoverflow.com/questions/65653226/jest-and-randombytesrequest-open-handles.